src/helm/proxy/clients/critique_client.py

```diff
@@ -209,3 +217,239 @@ def make_critique_request(self, request: CritiqueRequest) -> CritiqueRequestResu
+    def _get_or_create_scale_project(self, template: CritiqueTaskTemplate) -> str:
+        """Get or create a project on Scale and return the Scale project name.
+
+        Attempt to find a Scale project for the template. If one exists, reuse that project.
+        Otherwise, create a new project using the template. Return the Scale project name."""
+
+        def create_scale_project():
+            try:
+                project = self.client.create_project(
+                    project_name=template.name,
+                    task_type=TaskType.TextCollection,
+                    params={
+                        "instructions": template.instructions,
+                    },
+                )
+            except ScaleDuplicateResource as err:
+                hlog(f"ScaleDuplicateResource when creating project: {template.name}. Error: {err.message}")
+                # Get the existing project and checks that it has the same instructions
+                # NOTE: This should not happen with the cache but in case the cache is deleted
+                # we want to make sure we don't create a new project with the same name
+                project = self.client.get_project(template.name)
+                if project.params["instructions"] != template.instructions:
+                    raise RuntimeError(
+                        f"Project {template.name} already exists with different instructions: "
+                        f"{project.params['instructions']}"
+                    ) from err
+                elif project.type != TaskType.TextCollection.value:
+                    raise RuntimeError(
+                        f"Project {template.name} already exists with different task_type: '{project.type}' instead of '{TaskType.TextCollection.value}'"
+                    ) from err
+            return {"name": project.name}
+
+        with _scale_cache_lock:
+            # Example cache key:
+            # {
+            #   "name": "some_name",
+            #   "instructions": "some_instructions"
+            # }
+            #
+            # Example cache value:
+            # {"name": "some_name"}
+            project_response, is_cached = self._cache.get(
+                {"name": template.name, "instructions": template.instructions}, create_scale_project
+            )
+        project_name = project_response["name"]
+        if is_cached:
+            hlog(f"Reusing existing Scale project: {project_name}")
+        else:
+            hlog(f"Creating new Scale project: {project_name}")
+        return project_name
+
+    def _interpolate_fields(self, text: str, fields: Dict[str, str]) -> str:
+        for field_name, field_value in fields.items():
+            text = text.replace(f"{{{field_name}}}", field_value)
+        return text
+
+    def _critique_question_to_scale_field(self, question: CritiqueQuestionTemplate, fields: Dict[str, str]):
+        if question.question_type == "multiple_choice" or question.question_type == "checkbox":
```

Comment:
I think you have to set `min_choices` = `max_choices` = 1 for `multiple_choice` and `min_choices` = 1 and `max_choices` = `len(options)` for `checkbox`. ([link](https://docs.scale.com/reference/categoryfield))

Reply:
Oh you are right, thanks